### PR TITLE
Fixed import statement, code clean up and fixed console warnings

### PR DESCRIPTION
**Pull request by SilviuOdobescu**

### CODE REVIEW

File: AlfrescoApp/Utils/Categories/AlfrescoNode+Sync.h [hunk 1]
```diff
@@ -16,7 +16,8 @@
  *  limitations under the License.
  ******************************************************************************/
 
-#import "AlfrescoNode.h"
+#import <AlfrescoSDK-iOS/AlfrescoSDK.h>
```
Comment (mikeh) [inline AlfrescoApp/Utils/Categories/AlfrescoNode+Sync.h]: I'm surprised this is needed. The `AlfrescoSDK.h` file should be included as part of the precompiled header.

Comment (SilviuOdobescu) [inline AlfrescoApp/Utils/Categories/AlfrescoNode+Sync.h]: I believe that the initial import is what caused the build to fail.

Comment (mikeh) [inline AlfrescoApp/Utils/Categories/AlfrescoNode+Sync.h]: Sorry, I mean I'm surprised an import is needed at all.

Comment (SilviuOdobescu) [inline AlfrescoApp/Utils/Categories/AlfrescoNode+Sync.h]: You are right, it is not needed as I just performed additional checks. Will remove the import.

**Pull request merged**